### PR TITLE
feat(remote): adds new one and loads it in remote 1 and on the host

### PR DESCRIPTION
**Pull request by urecio**

### CODE REVIEW

File: webpack-remote/src/components/app.jsx [hunk 1]
```diff
@@ -30,6 +39,11 @@ export default function App({ chunks }) {
         {chunks.map((c) =>
           c.endsWith(".js") ? <script key={c} src={`build/${c}`} /> : null
         )}
+
+        <script
+          key="webpackRemote2_url"
+          src={`http://localhost:3003/build/remote-entry.js`}
```
Comment (AlzAmido) [inline webpack-remote/src/components/app.jsx:45]: we probably want to add the .env stuff to all remotes too

File: webpack-host/webpack.config.js [hunk 2]
```diff
@@ -7,6 +7,22 @@ const webpack = require("webpack");
 const nodeExternals = require("webpack-node-externals");
 // const { StatsWriterPlugin } = require("webpack-stats-plugin");
 const packageJsonDeps = require("./package.json").dependencies;
+const dotenv = require("dotenv").config();
+
+const WEBPACK_REMOTE_URL = dotenv.WEBPACK_REMOTE_URL || "http://localhost:3001";
+const WEBPACK_REMOTE_2_URL =
+  dotenv.WEBPACK_REMOTE_2_URL || "http://localhost:3003";
+
+const REMOTE_HOSTS = {
```
[the remark this answers is not in the document]
Comment (urecio) [reply webpack-host/webpack.config.js:16]: They come from an .env file already.
```const WEBPACK_REMOTE_2_URL =
  dotenv.WEBPACK_REMOTE_2_URL || "http://localhost:3003";```

File: webpack-remote/src/components/app.jsx (hunk 1 above)
Comment (urecio) [reply webpack-remote/src/components/app.jsx:45]: ok will do

File: webpack-host/webpack.config.js [hunk 3]
```diff
@@ -7,6 +7,22 @@ const webpack = require("webpack");
 const nodeExternals = require("webpack-node-externals");
 // const { StatsWriterPlugin } = require("webpack-stats-plugin");
 const packageJsonDeps = require("./package.json").dependencies;
+const dotenv = require("dotenv").config();
+
+const WEBPACK_REMOTE_URL = dotenv.WEBPACK_REMOTE_URL || "http://localhost:3001";
```
[the remark this answers is not in the document]
Comment (urecio) [reply webpack-host/webpack.config.js:9]: They are, to set the remotes down bellow

**Pull request merged**